src/backend/src/app.config.ts

```diff
@@ -46,7 +46,9 @@ export async function createNestApp(): Promise<{
     res.header('Access-Control-Allow-Headers', 'Content-Type, Accept');
     next();
   });
-  app.enableCors();
+  app.enableCors({
+    origin: ['http://localhost:3000', 'http://localhost:8080'],
```

Comment:
Can you try without 8080 one? It should work 🤔

Reply:
sure, i will do test later, if it works, i will remove 'http://localhost:8080'

Reply:
remove it